starter_scripts/mount_point_parsing.sh

```diff
@@ -124,6 +124,7 @@ function _format_parameters() {
 
 declare -a mount_point_paths
 formatted_params=$(_format_parameters "${@}")
+# shellcheck disable=SC2086
```

Comment:
Must not use quotes for line below, formatted_params  needs to be split by spaces (Intentionally)

Reply:
maybe add a comment, why this is safe

Reply:
Done